packages/flame/lib/src/effects/glow_effect.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:ui';
+
+import 'package:flame/effects.dart';
+import 'package:flame/src/effects/provider_interfaces.dart';
+
+/// Change the MaskFilter on Paint of a component over time.
+///
+/// This effect applies incremental changes to the MaskFilter on Paint of a
+/// component and
+/// requires that any other effect or update logic applied to the same component
+/// also used incremental updates.
```

Comment:
These lines should be formatted so that they fill up to 80 chars and then line break, not break in the middle of a line.